edit/edit.css

```diff
@@ -440,6 +471,10 @@ body[data-match-highlight="selection"] .CodeMirror-selection-highlight-scrollbar
 }
 #lint td[role="message"] {
   text-align: left;
+  max-width: 140px;
```

Comment:
BTW avoid changing the historical layout/appearance which isn't definitely buggy without discussion.